fastqheat/__main__.py

```diff
@@ -60,194 +54,199 @@ def _make_accession_list(term: str) -> list[str]:
+    return re.split('[ ,]+', value)
+
+
+@tp.no_type_check
+def validate_config(ctx, param, value) -> configparser.ConfigParser:
+    config = configparser.ConfigParser()
+    config.read(value)
+    if 'SRA' not in config.keys():
+        raise click.BadParameter(f"SRA section not found in config {value}")
+    if 'SSHKey' not in config['SRA'].keys():
+        raise click.BadParameter(f"SSHKey not found in SRA section of config {value}")
+    if 'FasterQDump' not in config['SRA'].keys():
+        raise click.BadParameter(f"FasterQDump not found in SRA section of config {value}")
+    if 'ENA' not in config.keys():
+        raise click.BadParameter(f"ENA section not found in config {value}")
+    if 'AsperaFASP' not in config['ENA'].keys():
+        raise click.BadParameter(f"AsperaFASP not found in ENA section of config {value}")
+    return config
+
+
+def common_options(f: tp.Callable) -> tp.Callable:
+    f = click.option(
+        '--working-dir',
+        default=os.getcwd,
+        type=click.Path(exists=True, file_okay=False, dir_okay=True, writable=True),
+        show_default=True,
+        help='Working directory.',
+    )(f)
+    f = click.option(
+        '--config',
+        default=get_config_path,
+        type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+        callback=validate_config,
+        show_default=True,
+        help='Configuration file path.',
+    )(f)
+    f = click.option(
+        '--accession',
+        default=[],
+        show_default=True,
+        callback=validate_accession,
+        help='List of accessions separated by comma. E.g "111,222,333"',
+    )(f)
+    f = click.option(
+        '--attempts',
+        default=0,
+        show_default=True,
+        help='Retry attempts in case of network error.',
+        type=click.IntRange(min=0),
+    )(f)
+    f = click.option(
+        '--attempts_interval',
+        default=0,
+        show_default=True,
+        help='Retry attempts interval in seconds in case of network error.',
+        type=click.IntRange(min=0),
+    )(f)
+    f = click.option(
+        '--skip-download',
+        default=False,
+        show_default=True,
+        help='Skip data download step. Data check (if not skipped) will '
+        'expect data to be in the working directory',
+        type=click.BOOL,
+    )(f)
+    f = click.option(
+        '--skip-check',
+        default=False,
+        show_default=True,
+        help='Skip data check step.',
+    )(f)
+    f = click.option(
+        '--cpu-count',
+        default=get_cpu_cores_count,
+        show_default=True,
+        help='Number of binaries or data checking threads to be working simultaneously.',
+        type=click.IntRange(min=1),
+    )(f)
+    return f
+
+
+def get_config_path() -> str:
+    return os.path.join(os.path.dirname(__file__), 'config.conf')
+
+
+def get_metadata_file() -> str:
+    return os.path.join(os.getcwd(), 'metadata.csv')
```

Comment:
`Path().cwd() / 'metadata.csv'` ? 

I mean, `pathlib.Path` is much more concise and convenient way of working with paths, it will be nice to use it instead of cumbersome `os.path`